### PR TITLE
Topic recognition - ISIC 2018 Segmentation with UNet

### PR DESCRIPTION
**Pull request by ghost**
Name: Jaydon Hansen
Student number: 45875683

Image Segmentation of ISIC 2018 data with UNet. Please let me know if you want me to provide the resulting .h5 file from the training.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Easy Difficulty
- Results are good

Other Feedback
- Good code structure / modularisation
- (OPTIONAL) Would put main logic in `if __name__ == '__main__'` for the sake of following python convention.

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;


Tensorflow has functions for image resizing and more so there’s no need to also use PIL in your project. Same with numpy and array manipulation

Comments such as 
      # Compile a model and show the shape
      model.compile(optimizer="adam", loss="binary_crossentropy", metrics=[dice_coef, "acc"])
Add nothing to the readability of the code and should be avoided. Overall comments were well done

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
